apps/backend/src/datasources/mockups/FapDataSource.ts

```diff
@@ -173,6 +173,12 @@ export class FapDataSourceMock implements FapDataSource {
   async assignChairOrSecretaryToFap(
     args: AssignChairOrSecretaryToFapInput
   ): Promise<Fap> {
+    if (args.userId === 5 || args.userId === 6) {
+      throw new Error(
+        'Can not assign to Fap, because only users with fap reviewer role can be chair or secretary'
+      );
+    }
```

Comment:
suggestion: I have tried running the tests with these lines commented out and the tests are still passing. Are you sure we need these lines?